components/logins/ios/Logins/LoginsStorage.swift

```diff
@@ -8,14 +8,8 @@ import UIKit
 
 typealias LoginsStoreError = LoginsStorageError
 
-/*
- ** We probably should have this class go away eventually as it's really only a thin wrapper
- * similar to its kotlin equiavlent, however the only thing preventing this from being removed is
- * the queue.sync which we should be moved over to the consumer side of things
- */
```

Comment:
Do you think we should eventually try and remove this file? It looks like now it's really only the `sync` method which does some arg massaging. If you think that's still a reasonable idea, maybe keep a comment here saying that's still something we should consider?

Reply:
Wondering if this or something similar can live in Firefox iOS? 

Does A~S team uses this class for testing any Rust component? 

cc @lougeniaC64